src/polyglot_ffi/generators/python_gen.py

```
@@ -218,3 +273,271 @@ def _get_ctypes(self, ir_type: IRType) -> str:
+    ) -> list:
+        """Generate code to convert Python nested list (List[List[T]]) to C array of arrays."""
+        lines = []
+        param_name = param.name
+
+        if not inner_list_type.params or not inner_list_type.params[0].is_primitive():
+            # Only support primitive inner types
+            return lines
+
+        inner_element_type = inner_list_type.params[0]
+
+        lines.append(f"        # Convert Python nested list to C array of arrays")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)